bitrise.yml

```diff
@@ -26,6 +26,13 @@ workflows:
             #!/usr/bin/env bash
             set -ex
             ./gradlew :arrow-docs:runAnk
+    - script:
+        title: check
+        inputs:
+          - content: |-
+              #!/usr/bin/env bash
+              set -ex
+              ./gradlew check
```

Comment:
I'd suggest adding either bitrise or github actions, but not both. @rachelcarmena Do you know which one should we use? I'd personally go for github actions as it seems it runs faster :)

Reply:
Ok, I will revert the bitrise config file

Reply:
Thanks for the mention @aballano. It would be just necessary to update GitHub Actions. because we have the build check there:
- Travis CI is only used to publish snapshots/releases
- Bitrise is only used to generate the documentation

Thanks!

Reply:
Reverted